st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -43,6 +43,9 @@
 ]
 
 LOG = logging.getLogger(__name__)
+LOG_PATH = '/var/log/st2/'
```

Comment:
The reason you couldn't find one is because the log path is customizable via the config files mentioned in my comment above. It is not hardcoded to /var/log/st2.